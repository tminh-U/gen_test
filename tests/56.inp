11
16 9 10 10 20 3 11 12 12 2 6 
15
8 9
11 11
10 10
10 11
7 9
2 5
5 9
8 8
1 4
9 10
5 7
7 11
9 9
1 1
5 8
