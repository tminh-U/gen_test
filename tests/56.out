4
2
2
4
8
16
17
2
9
3
5
11
2
2
9
